api/TextTrackCue.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "≤79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "42"
```

Comment:
It is odd if `TextTrackCue` shipped after `VTTCue`, since `VTTCue` inherits from `TextTrackCue`. But `TextTrackCue` is an abstract interface, so it may in fact have happened.

Can you also update https://developer.mozilla.org/en-US/docs/Web/API/VTTCue as part of this, or better yet split out the text track-related stuff Text* and VTT* into a separate PR?